packages/front-end/components/FactTables/FactMetricList.tsx

```diff
@@ -34,13 +34,15 @@ export function getMetricsForFactTable(
 }
 
 export default function FactMetricList({ factTable }: Props) {
-  const [editOpen, setEditOpen] = useState("");
```

Comment:
This didn't seem like it was being used. Nothing was setting `editOpen` so I removed this state